File: core/src/main/java/vn/loitp/flickr/model/photosetgetlist/Description.java

package vn.loitp.flickr.model.photosetgetlist;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Description {

    @SerializedName("_content")
    @Expose
    private String content;

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

}

File: core/src/main/java/com/loitpcore/restApi/flickr/model/photoSetGetList/Description.kt
package com.loitpcore.restApi.flickr.model.photoSetGetList

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.loitpcore.core.base.BaseModel

@Keep
class Description : BaseModel() {
    @SerializedName("_content")
    @Expose
    var content: String? = null
}
